<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Судоку</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>490</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Проверить</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>490</y>
      <width>351</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_5">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_6">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_8">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>120</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_9">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_10">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_11">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_12">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_13">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_14">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>70</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_15">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>70</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_16">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>120</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_17">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>120</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_18">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>120</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_19">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_20">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>20</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_21">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>20</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_22">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>70</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_23">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>70</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_24">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>70</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_25">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>120</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_26">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>120</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_27">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>120</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_28">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_29">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>170</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_30">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>170</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_31">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_32">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>220</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_33">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>220</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_34">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_35">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>270</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_36">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>270</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_37">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>170</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_38">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>170</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_39">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>170</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_40">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>220</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_41">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>220</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_42">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>220</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_43">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>270</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_44">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>270</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_45">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>270</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_46">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>170</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_47">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>170</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_48">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>170</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_49">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>220</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_50">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>220</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_51">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>220</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_52">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>270</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_53">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>270</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_54">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>270</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_55">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_56">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>320</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_57">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>320</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_58">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_59">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>370</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_60">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>370</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_61">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_62">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>420</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_63">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>420</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_64">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>320</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_65">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>320</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_66">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>320</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_67">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>370</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_68">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>370</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_69">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>370</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_70">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>420</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_71">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>420</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_72">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>420</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_73">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>320</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_74">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>320</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_75">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>320</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_76">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>370</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_77">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>370</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_78">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>370</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_79">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>420</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_80">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>420</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_81">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>420</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>520</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Назад</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>520</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Подсказка</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>153</x>
      <y>20</y>
      <width>41</width>
      <height>451</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>41</width>
      <height>451</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>41</width>
      <height>451</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>443</x>
      <y>20</y>
      <width>61</width>
      <height>451</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>451</width>
      <height>21</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>451</width>
      <height>21</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>451</width>
      <height>21</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>455</y>
      <width>451</width>
      <height>31</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>520</y>
      <width>281</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>498</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
